src/main/webapp/app/components/exercise/presentation-score/presentation-score.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+
+import { ArtemisSharedModule } from 'app/shared';
+import { PresentationScoreComponent } from 'app/components/exercise/presentation-score/presentation-score.component';
+
+@NgModule({
+    imports: [ArtemisSharedModule],
+    declarations: [PresentationScoreComponent],
+    exports: [PresentationScoreComponent],
```

Comment:
Can this go into the `ArtemisSharedModule`? Having its own module is a bit overkill.

Reply:
Thank you for the review :) yes you're right - I changed that!